cli/aws_orbit/data/toolkit/template.yaml

```diff
@@ -196,15 +196,15 @@ Resources:
             Action:
               - redshift:*
             Resource:
-              - arn:aws:redshift:${region}:${account_id}:parametergroup:${top_level}-${env_name}-*
-              - arn:aws:redshift:${region}:${account_id}:securitygroup:${top_level}-${env_name}-*
-              - arn:aws:redshift:${region}:${account_id}:subnetgroup:${top_level}-${env_name}-*
-              - arn:aws:redshift:${region}:${account_id}:cluster:${top_level}-${env_name}-*
+              - arn:aws:redshift:${region}:${account_id}:parametergroup:orbit-${env_name}-*
```

Comment:
We use boto3 to delete the leftover redshift cluster(s) before deleting the redshift plugin resources via codebuild and respective orbit admin role. The redshift cluster starts with {env_name} and related parametergroups, etc starts with orbit instead of orbit-foundation. If we have to be flexible then need to use the same top_level variable value while creating redshift cluster via lambda.